lib/tapioca/rbi/model.rb

```diff
@@ -390,6 +396,12 @@ def initialize(name, loc: nil, comments: [])
       def to_s
         name
       end
+
+      sig { params(other: T.nilable(Object)).returns(T::Boolean) }
+      def ==(other)
+        return false unless other.instance_of?(Param)
+        name == T.cast(other, Param).name
```

Comment:
Side note, couldn't `instance_of?` be supported in Sorbet's flow typing?

Reply:
There is already an issue for that and it's less trivial than what it seems https://github.com/sorbet/sorbet/issues/48